chainer/links/connection/lstm.py

```diff
@@ -27,14 +53,31 @@ class LSTM(link.Chain):
         h (~chainer.Variable): Output at the previous time step.
 
     """
-    def __init__(self, in_size, out_size):
+
+    def __init__(self, in_size, out_size,
+                 lateral_init=None, upward_init=None,
+                 bias_init=0, forget_bias_init=0):
         super(LSTM, self).__init__(
-            upward=linear.Linear(in_size, 4 * out_size),
-            lateral=linear.Linear(out_size, 4 * out_size, nobias=True),
+            upward=linear.Linear(in_size, 4 * out_size, initialW=0),
```

Comment:
No, it is not. However, I would consider the original a bug. The scaling should be based on each unit of the LSTM. The original would adjust the scaling as if there were 4 times as much fan out.